### PR TITLE
UIOR-507 refactoring PO Lines list to not use SearchAndSort

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**
## Purpose
TECH-DEBT refactor PO Lines list to not use SearchAndSort
https://issues.folio.org/browse/UIOR-507

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/OrderLinesList/OrderLinesListContainer.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    setOrderLinesOffset(0);
+    loadOrderLines(0);
+  };
+
+  useEffect(
+    refreshList,
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [location.search],
+  );
+  useLocationReset(history, location, '/invoice', refreshList);
```
Comment (NikitaSedyx) [inline src/OrderLinesList/OrderLinesListContainer.js]: `/orders/lines`?

File: src/OrderLinesList/OrderLinesFiltersContainer.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+} from '@folio/stripes-acq-components';
+
+import {
+  MATERIAL_TYPES,
+} from '../components/Utils/resources';
+import OrderLinesFilters from './OrderLinesFilters';
+
+const OrderLinesFiltersContainer = ({ resources, activeFilters, applyFilters }) => {
+  const materialTypes = get(resources, 'materialTypes.records', []);
+  const funds = get(resources, `${DICT_FUNDS}.records`, []);
```
Comment (uladislausamets) [inline src/OrderLinesList/OrderLinesFiltersContainer.js]: Can we don't use resources and load data manually?

Comment (aliaksei-chumakou) [reply src/OrderLinesList/OrderLinesFiltersContainer.js]: yes, we can, but it's not the point of the PR. Also it's pretty improtant to understand the difference of _load data manually_ and stripes-connect `fetch: true` resources. Stripes-connect adds kind of cache layer, so we don't need to make actual calls on back-end for dictionary data. In this particular case all funds are loading, that is wrong by the meaning. In future all such places will be changed to look-up plugins, for now as temporary solution we try to use that cache thing from stripes-connect

Comment (uladislausamets) [reply src/OrderLinesList/OrderLinesFiltersContainer.js]: Thnx for clarification)

**Pull request merged**